pkg/controllers/scheduling/scheduling_controller.go

```diff
@@ -174,7 +181,10 @@ func (c *schedulingController) sync(ctx context.Context, syncCtx factory.SyncCon
 		return err
 	}
 
-	// update placement status if necessary
+	// update placement status if necessary to signal no bindings
+	if !bindingsFound {
+		scheduleResult.unscheduled = -1
+	}
 	return c.updateStatus(ctx, placement, scheduleResult.scheduled, scheduleResult.unscheduled)
```

Comment:
how about getting binding above, and change this func to 

c.updateStatus(ctx, placement, bindings, scheduleResult.scheduled, scheduleResult.unscheduled)

Then we can set conditions  when binding is empty clearer.

Reply:
That's fine. I was thinking about passing the param, but didn't want to modify the switch in the status.